toonz/sources/CMakeLists.txt

```diff
@@ -198,13 +219,65 @@ else()
 
     find_package(LZO REQUIRED)
     message("LZO:" ${LZO_INCLUDE_DIR})
+
+elseif(UNIX)
+    find_library(GL_LIB GL)
+    find_library(GLU_LIB GLU)
+    find_library(GLUT_LIB glut)
+    find_library(Z_LIB z)
+
+    find_package(TIFF REQUIRED)
+    set(TIFF_LIB ${TIFF_LIBRARY})
+    # the libraries have not .pc nor preferred Find*.cmake. use custom modules.
+    # find_package(PNG REQUIRED)
+    set(PNG_LIB ${SDKROOT}/libpng-1.6.21/.libs/libpng16.so)
```

Comment:
Being explicit about dependencies within the CMake files can make bisecting older versions of the software difficult. (unless your project keeps an image of the platform used to build each version - some do).
In my experience its best to store exact configurations in a separate CMake file or build script.